### PR TITLE
Bug/timeseries fixes

### PR DESCRIPTION
**Pull request by ramneet-persistent**
### Issues Resolved
[timeseries/scatter bugs](https://github.com/opensearch-project/observability/issues/1104)

### CODE REVIEW

File: dashboards-observability/common/constants/explorer.ts [hunk 1]
```diff
@@ -233,6 +234,28 @@ export const DEFAULT_STATS_CHART_PARAMETERS: DefaultStatsChartParametersProps =
...
+
+export const DEFAULT_BAR_CHART_STYLES: DefaultBarChartStylesProps = {
+  BarMode: 'group',
+  GroupWidth: 0.7,
+  BarWidth: 0.97,
+  LineWidth: 1,
+  LabelSize: 12,
+};
+
+export const SIMILAR_VIZ_TYPES = [
```
Comment (ruchika-narang) [inline dashboards-observability/common/constants/explorer.ts:253]: Check if this object is needed now, else remove this.

File: dashboards-observability/public/components/visualizations/charts/lines/line.tsx [hunk 2]
```diff
@@ -79,10 +74,11 @@ export const Line = ({ visualizations, layout, config }: any) => {
   const labelSize = dataConfig?.chartStyles?.labelSize;
   const legendSize = dataConfig?.legend?.legendSize;
 
+  console.log('labelSize===', labelSize);
```
Comment (ruchika-narang) [inline dashboards-observability/public/components/visualizations/charts/lines/line.tsx]: Remove console.

**Review by ruchika-narang — CHANGES_REQUESTED** [review]
Please address the comments.

File: dashboards-observability/public/components/visualizations/charts/bar/bar_type.ts [hunk 3]
```diff
@@ -8,42 +8,50 @@ import { getPlotlySharedConfigs, getPlotlyCategory } from '../shared/shared_conf
...
 export const createBarTypeDefinition = (params: any) => ({
-  name: 'bar',
+  name: params.type || 'bar',
   type: 'bar',
-  id: 'bar',
-  label: 'Vertical bar',
-  fulllabel: 'Vertical bar',
-  icontype: 'visBarVerticalStacked',
+  id: params.type || 'bar',
+  label: isHorizontalBar(params.type) ? 'Horizontal bar' : 'Vertical bar',
```
Comment (spattnaik) [inline dashboards-observability/public/components/visualizations/charts/bar/bar_type.ts:38]: isHorizontalBar(params.type) can be taken into a variable, as being used 3-4 times

File: dashboards-observability/public/components/visualizations/charts/lines/line.tsx (hunk 2 above)
Comment (spattnaik) [inline dashboards-observability/public/components/visualizations/charts/lines/line.tsx]: can be removed

**Review by spattnaik — CHANGES_REQUESTED** [review]
Minor changes

File: dashboards-observability/common/constants/explorer.ts (hunk 1 above)
Comment (ramneet-persistent) [reply dashboards-observability/common/constants/explorer.ts:253]: yes it is used in bar.tsx as default value if not applied in chart styles

File: dashboards-observability/public/components/visualizations/charts/bar/bar_type.ts (hunk 3 above)
Comment (ramneet-persistent) [reply dashboards-observability/public/components/visualizations/charts/bar/bar_type.ts:38]: @spattnaik
 isHorizontalBar(params.type) declared separately and its boolean variable but usecases are different

**Review by spattnaik — APPROVED** [review]

**Review by ruchika-narang — APPROVED** [review]

**Comment by ramneet-persistent** [comment]
External [PR #1113](https://github.com/opensearch-project/observability/pull/1113) raised
so closing this one

**Pull request closed without merging**